features/locale/handling_different_locales_on_multiple_channels.feature

```diff
@@ -1,33 +1,33 @@
 @locales
 Feature: Handling different locales on multiple channels
-    In order to see right prices
+    In order to present shop data in a correct language
     As a Customer
-    I want to browse channels with a valid locale only
+    I want to browse channels with a valid locale
 
     Background:
-        Given the store operates on a channel named "Web"
+        Given the store operates on a channel named "Web" with hostname "web.example"
         And that channel allows to shop using "English (United States)", "Polish (Poland)" and "Norwegian (Norway)" locales
         And it uses the "English (United States)" locale by default
-        And the store operates on another channel named "Mobile"
+        And the store operates on another channel named "Mobile" with hostname "mobile.example"
         And that channel allows to shop using "Polish (Poland)" and "Norwegian (Norway)" locales
         And it uses the "Polish (Poland)" locale by default
 
     @ui
     Scenario: Showing locales only from the current channel
         When I browse the "Mobile" channel
-        Then I should shop using the "polski (Polska)" locale
+        Then I should shop using the "Polish (Poland)" locale
         And I should be able to shop using the "norweski (Norwegia)" locale
         And I should not be able to shop using the "angielski (Stany Zjednoczone)" locale
```

Comment:
These ones should also be in English to make steps consistent

Reply:
The tricky part here is that we have a non-polish locale with polish translation. So then I would have to do something like `:localNameInPolishLocale` :thinking:

Reply:
I did `:localeNameInCurrentLocale` so `:localeNameInItsLocale` is now redundant :slightly_smiling_face: